docs/docs/config/battery.md

```diff
@@ -55,7 +55,7 @@ Driver for reading the voltage of a battery using an ADC connected to a voltage
 
 Applies to: `compatible = "zmk,battery-voltage-divider"`
 
-See [Zephyr's voltage divider documentation](https://docs.zephyrproject.org/3.5.0/build/dts/api/bindings/iio/afe/voltage-divider.html).
+See [Zephyr's voltage divider documentation](https://docs.zephyrproject.org/3.5.0/build/dts/api/bindings/iio/afe/voltage-divider.html). <!-- codespell:ignore afe -->
```

Comment:
Does this not ignore URL's when checking spelling?

Reply:
Nope, it error'ed

Reply:
It looks like it is possible to configure it to ignore them: https://github.com/codespell-project/codespell/blob/main/codespell_lib/_codespell.py#L471

I think it's a good idea to ignore them, if anyone else thinks otherwise let me know.